kyaml/setters2/settersutil/substitutioncreator.go

```diff
@@ -77,7 +77,7 @@ func (c SubstitutionCreator) Create(openAPIPath, resourcesPath string) error {
 
 // CreateSettersForSubstitution creates the setters for all the references in the substitution
 // values if they don't already exist in openAPIPath file.
-func (c SubstitutionCreator) CreateSettersForSubstitution(openAPIPath string) error {
+func (c *SubstitutionCreator) CreateSettersForSubstitution(openAPIPath string) error {
```

Comment:
Why do we need a pointer receiver for this function, but not for the others?

Reply:
Not needed and removed.